classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
```

Comment:
- I would advise you to declare the variable `students` before using it, because it is the best practice in Ruby. 
Hint: `attr_reader :students`
You can learn more about it from this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor%20#%20*Check%20that%20the%20instance%20variables%20have%20setters%20and%20getters)